### PR TITLE
Use translate method instead of accessing the lang string directly

### PR DESCRIPTION
**Pull request by simonschaufi**
Changes proposed in this pull request:
- use translate method instead of accessing the lang string directly

Reason for this pull request:
- Improve the translation handling while displaying untranslated strings when they are not available

### CODE REVIEW

File: extensions/ki_adminpanel/templates/scripts/floaters/editglobalrole.php [hunk 1]
```diff
@@ -9,23 +9,23 @@
...
                     <span class="cc">&nbsp;</span>
                 </a></li>
             <?php
             foreach ($keyHierarchy as $key => $subKeys):
                 if (count($subKeys) == 1 && array_key_exists('access', $subKeys)) continue;
 
                 $name = $key;
-                if (isset($this->kga['lang']['extensions'][$name]))
-                    $name = $this->kga['lang']['extensions'][$name];
+                if (isset($this->translate('extensions')[$name]))
```
Comment (simonschaufi) [inline extensions/ki_adminpanel/templates/scripts/floaters/editglobalrole.php]: does't work

File: extensions/ki_invoice/templates/scripts/floaters/editVat.php [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
             <fieldset>
-                <label for="vat"><?php echo $this->kga['lang']['vat'] ?></label>
+                <label for="vat"><?php echo $this->translate('vat') ?></label>
                 <input type="number" name="vat" id="vat" value="<?php echo $this->escape(str_replace('.', $this->kga['conf']['decimalSeparator'], $this->kga->getDefaultVat())); ?>" style="width: 50px;"/>
                 %
                 <div id="formbuttons">
-                    <input class='btn_norm' type='button' value='<?php echo $this->kga['lang']['cancel'] ?>' onclick='floaterClose();return false;'/>
-                    <input class='btn_ok' type='submit' value='<?php echo $this->kga['lang']['submit'] ?>'/>
+	                <button type="button" class="btn_norm" onclick="floaterClose();"><?php echo $this->translate('cancel') ?></button>
+	                <input type="submit" class="btn_ok" value="<?php echo $this->translate('submit') ?>" onclick="floaterClose();"/>
```
Comment (simonschaufi) [inline extensions/ki_invoice/templates/scripts/floaters/editVat.php]: - `floaterClose`

File: extensions/ki_timesheets/templates/scripts/floaters/add_edit_timeSheetQuickNote.php [hunk 3]
```diff
@@ -33,34 +33,34 @@
...
                         <?php echo $this->commentTypeSelect($this->commentType); ?>
                     </li>
                 </ul>
             </fieldset>
         </div>
         <div id="formbuttons">
-            <input class='btn_norm' type='button' value='<?php echo $this->kga['lang']['cancel'] ?>' onclick='floaterClose();return false;'/>
-            <input class='btn_ok' type='submit' value='<?php echo $this->kga['lang']['submit'] ?>'/>
+	        <input type="button" class="btn_norm" value="<?php echo $this->translate('cancel') ?>" onclick="floaterClose();return false;"/>
+	        <input type="submit" class="btn_ok" value="<?php echo $this->translate('submit') ?>" onclick="floaterClose();"/>
```
Comment (simonschaufi) [inline extensions/ki_timesheets/templates/scripts/floaters/add_edit_timeSheetQuickNote.php]: - `floaterClose`

**Comment by kevinpapst** [comment]
Why the hurry? Please leave the PR open for some time

**Comment by kevinpapst** [comment]
nevertheless, this is a team project and don't tell me that nobrainer don't cause bugs ;-)

**Pull request merged**